lib/sort-order-selector/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { toggleSortOrder, setSortType } from '../state/settings/actions';
+import classNames from 'classnames';
+import IconButton from '../icon-button';
+import SortOrderIcon from '../icons/sort-order';
+
+import * as S from '../state';
+import * as T from '../types';
+
+type StateProps = {
+  shouldDisplay: boolean;
+  sortReversed: boolean;
+  sortType: T.SortType;
+};
+
+type DispatchProps = {
+  toggleSortOrder: () => any;
+  setSortType: (sortType: T.SortType) => any;
+};
+
+type Props = StateProps & DispatchProps;
+
+export class SortOrderSelector extends Component<Props> {
```

Comment:
you may have already made this a function and moved between function and class but it looks like we could go ahead and make it a function component as we don't rely on any state. no need of course but this isn't depending on instance properties.